samtranslator/internal/schema_source/aws_serverless_graphqlapi.py

```diff
@@ -25,9 +25,10 @@ class DeltaSync(BaseModel):
 
 
 class DynamoDBDataSource(BaseModel):
-    # TableArn, Permissions properties for connector
-    ServiceRoleArn: PassThroughProp  # TODO: make optional when we ship connector changes
     TableName: str
+    ServiceRoleArn: Optional[PassThroughProp]
+    TableArn: Optional[str]
+    Permissions: Optional[List[str]]
```

Comment:
Should this be one of:

https://github.com/aws/serverless-application-model/blob/c29f64a069ead2658a2e5412df53002b85bde3ee/samtranslator/internal/schema_source/aws_serverless_connector.py#L10

Reply:
Yep I think thats more correct, let me use that typing instead